txpool/pool_test.go

```diff
@@ -823,13 +827,13 @@ func TestBlobTxReplacement(t *testing.T) {
 		}
 	}
 
-	//try to replace it with required extra gas - should pass\
+	//try to replace it with required price bump to all transaction fields - should be successful
```

Comment:
It'd be nice to add tests with individual bumps as well. At the moment it's not clear from the test what exactly should be bumped and by how much.